### PR TITLE
Add SwiftFormat to the codebase

### LINKED ISSUE #144: 🦺[Tech Debt] Add  Linter `Swift Format`
## Goal 
**Describe the work to be done**
 
`A small step for man a giant leap for mankind` - Armstrong

This code has needed a linter for years but the task was **huge**.  So many classes fail the linter (the one @kcw-grunt is familiar with).  Maybe fresh eyes can fix the issue.

## References
https://github.com/nicklockwood/SwiftFormat

```
let package = Package(
    name: "BuildTools",
    platforms: [.macOS(.v10_11)],
    dependencies: [
        .package(url: "https://github.com/nicklockwood/SwiftFormat", from: "0.49.0"),
    ],
    targets: [.target(name: "BuildTools", path: "")]
)
```

## Definition of Done
- [ ] Removes headers
- [ ] Auto to Xcode format standards (4 tab, \ n only between lines
- [ ] Other suggestions

### PR DESCRIPTION
**Pull request by Iferencak**
# Added `SwiftFormat` to the codebase

# format options
--allman true
--header strip
--indent tab

# file options
--exclude Pods

# rules
--enable isEmpty
--disable enumNamespaces

### CODE REVIEW

**Comment by kcw-grunt** [comment]
👀 ...looking @Iferencak

File: loafwallet.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -3729,13 +3729,13 @@
...
 				825805FFD443BACDA18265D0 /* [CP] Check Pods Manifest.lock */,
 				2430679A238F538C00EBEA99 /* Update Localizable using BartyCrouch */,
+				583FE8702927ED5A009A6384 /* Run SwiftFormat */,
 				75A2A78C1DA5934300A983D8 /* Sources */,
 				75A2A78D1DA5934300A983D8 /* Frameworks */,
 				75A2A78E1DA5934300A983D8 /* Resources */,
 				75A2A8031DA5935F00A983D8 /* Embed App Extensions */,
 				22A9A9831DF63288000F0016 /* Embed Frameworks */,
 				C3A3FFB526FE46E8000FE955 /* Mark Dev Notes */,
-				58A9FE4829191E4700B75825 /* Check for unused code */,
```
Comment (kcw-grunt) [inline loafwallet.xcodeproj/project.pbxproj:3738]: @Iferencak : So are you moving this functionality into the linter/formatter config?

File: loafwallet/AlertFailureView.swift [hunk 2]
```diff
@@ -1,60 +1,50 @@
...
-    let errorMessage: String
-    
-    init(alertFailureType: AlertFailureType, errorMessage: String) {
-        self.alertFailureType = alertFailureType
-        self.errorMessage = errorMessage
-    }
-    
-    var body: some View {
-        VStack {
-            Text(alertFailureType.header)
```
Comment (kcw-grunt) [inline loafwallet/AlertFailureView.swift:24]: The norm for indentation in my Xcode preferences has been 4 spaces...here we are going to 8. Is this intentional or a SwiftFormat default?  It looks cleaner just need to know to change my settings @Iferencak

File: loafwallet/AppDelegate.swift [hunk 3]
```diff
@@ -1,119 +1,93 @@
-//
```
Comment (kcw-grunt) [inline loafwallet/AppDelegate.swift:1]: All hail Swift Format 😇
 no need for a 25 line header

File: loafwallet.xcodeproj/project.pbxproj (hunk 1 above)
Comment (Iferencak) [reply loafwallet.xcodeproj/project.pbxproj:3738]: @kcw-grunt nice catch, It was accidentally removed, I added it back in.

File: loafwallet/AlertFailureView.swift (hunk 2 above)
Comment (Iferencak) [reply loafwallet/AlertFailureView.swift:24]: @kcw-grunt It looks a lot cleaner in my xCode, not sure why github presented it this way...

image

Comment (Iferencak) [reply loafwallet/AlertFailureView.swift:24]: But I see now if you try and higlight the empty space it just shows as one tab. So it should be ok.

File: loafwallet/BRAPIClient+KV.swift [hunk 4]
```diff
@@ -1,174 +1,177 @@
...
-                i += UInt(MemoryLayout<UInt64>.size)
-                let deleted = dat.uInt8(atOffset: i) > 0
-                i += UInt(MemoryLayout<UInt8>.size)
-                items.append((key, ver, date, deleted ? .tombstone : nil))
-                self.client.log("keys: \(key) \(ver) \(date) \(deleted)")
-            }
-            completionFunc(items, nil)
-        }.resume()
-    }
+private class KVStoreAdaptor: BRRemoteKVStoreAdaptor {
```
Comment (kcw-grunt) [inline loafwallet/BRAPIClient+KV.swift:17]: Started an issue where I'm 80% sure this code is ded: #163 
cc: @Iferencak

File: loafwallet/ContainerViewController.swift [hunk 5]
```diff
@@ -1,28 +1,17 @@
...
-
 import Foundation
 import UIKit
 
 class ContainerViewController: UIViewController {
-  
-  override func viewDidLoad() {
-    
-  }
+	override func viewDidLoad()
```
Comment (kcw-grunt) [inline loafwallet/ContainerViewController.swift:5]: Started an issue where I'm 80% sure this code is ded: https://github.com/litecoin-foundation/litewallet-ios/issues/163
cc: @Iferencak

File: loafwallet/Currency.swift [hunk 6]
```diff
@@ -1,60 +1,50 @@
...
-            case .idr: return 2
-            case .rub: return 3
-            case .jpy: return 4
-            case .eur: return 5
-            case .gbp: return 6
-            case .usd: return 7
-        }
-    }
-}
+	case cad
```
Comment (kcw-grunt) [inline loafwallet/Currency.swift:12]: This needs updating: #164

File: loafwallet/EmptyTableViewCell.swift [hunk 7]
```diff
@@ -1,14 +1,5 @@
...
-//  Copyright © 2021 Litecoin Foundation. All rights reserved.
-//
-
 import Foundation
 import UIKit
 
-class EmptyTableViewCell: UITableViewCell {
-
-}
+class EmptyTableViewCell: UITableViewCell
```
Comment (kcw-grunt) [inline loafwallet/EmptyTableViewCell.swift:4]: I'm not proud....Maybe It can be removed.

File: loafwallet/FeeSelectorView.swift [hunk 8]
```diff
@@ -1,101 +1,95 @@
...
 
+class FeeSelector: UIView {
+	init(store: Store) {
+		self.store = store
+		super.init(frame: .zero)
+		setupViews()
+	}
 
-class FeeSelector : UIView {
+	var didUpdateFee: ((FeeType) -> Void)?
```
Comment (kcw-grunt) [inline loafwallet/FeeSelectorView.swift:11]: I prefer to have declarations before init.  Alot of the legacy code has that swapprd and its really annoyingv I did this.  Not sure why I prefer it but I do. 

Normally make blocks of types..then init
The consistency is soothing....call me weird.

File: loafwallet/ForgotViewModel.swift [hunk 9]
```diff
@@ -1,12 +1,4 @@
-//
-//  ForgotViewModel.swift
-//  loafwallet
-//
-//  Created by Kerry Washington on 12/24/20.
-//  Copyright © 2020 Litecoin Foundation. All rights reserved.
-//
-
 import Foundation
```
Comment (kcw-grunt) [inline loafwallet/ForgotViewModel.swift:1]: Haha...The ForgottenViewModel (#163)

File: loafwallet/LockScreenHeaderView.swift [hunk 10]
```diff
@@ -1,65 +1,52 @@
...
+
+	@ObservedObject
+	var viewModel: LockScreenHeaderViewModel
+
+	init(viewModel: LockScreenHeaderViewModel) {
+		self.viewModel = viewModel
+	}
+
+	var body: some View {
+		Color
```
Comment (kcw-grunt) [inline loafwallet/LockScreenHeaderView.swift:14]: Ugh...I don't like this.
I seems unnatural like a Chucky Doll. I see what's happening though. Tab 8 spaces means short View names sit by themselves
cc: @Iferencak

File: loafwallet/LoginCardAlertView.swift [hunk 11]
```diff
@@ -1,86 +1,73 @@
...
+						RoundedRectangle(cornerRadius: 8)
+							.stroke(Color.gray, lineWidth: 1.5)
+					)
+					.background(Color.white)
+					.cornerRadius(8)
+					.frame(
+						width: deviceSize.size.width * 0.85,
+						height: deviceSize.size.height * 0.5
+					)
+					.shadow(color: .gray, radius: /*@START_MENU_TOKEN@*/10/*@END_MENU_TOKEN@*/, x: 5, y: 5)
```
Comment (kcw-grunt) [inline loafwallet/LoginCardAlertView.swift:53]: Curious: `/*@START_MENU_TOKEN@*/10/*@END_MENU_TOKEN@*/`

File: loafwallet/src/Views/PinPadCells/PinPadCells.swift [hunk 12]
```diff
@@ -1,9 +1 @@
-//
```
Comment (kcw-grunt) [inline loafwallet/src/Views/PinPadCells/PinPadCells.swift:1]: Ugh...adding to this ticket #163

File: loafwallet/src/Views/NonScrollingCollectionView.swift [hunk 13]
```diff
@@ -1,18 +1,10 @@
-//
-//  NonScrollingCollectionView.swift
-//  breadwallet
```
Comment (kcw-grunt) [inline loafwallet/src/Views/NonScrollingCollectionView.swift:3]: Can believe this ever got called. adding to #163

File: loafwallet/BRAPIClient+Events.swift [hunk 14]
```diff
@@ -1,267 +1,258 @@
-//
-//  BRAPIClient+Events.swift
-//  breadwallet
-//
-//  Created by Samuel Sutch on 7/4/17.
-//  Copyright © 2017 breadwallet LLC. All rights reserved.
-//
-
 import Foundation
```
Comment (kcw-grunt) [inline loafwallet/BRAPIClient+Events.swift:1]: **Wont** add this to a kill ticket but the architecture smells of their private cloud/log. But I think, its being called but not really used by us 
FYI: @Iferencak

File: loafwallet/PartnerAPIManager.swift [hunk 15]
```diff
@@ -1,20 +1,10 @@
-//
```
Comment (kcw-grunt) [inline loafwallet/PartnerAPIManager.swift:1]: Think this is deletable : #163

File: loafwallet/PartnersViewModel.swift [hunk 16]
```diff
@@ -1,16 +1,6 @@
-//
```
Comment (kcw-grunt) [inline loafwallet/PartnersViewModel.swift:1]: This is me going full tilt on injecting dependencies and then not testing or adding any  functionality in the model

File: loafwallet/SupportLFViewModel.swift [hunk 17]
```diff
@@ -1,15 +1,5 @@
-//
-//  SupportLFViewModel.swift
```
Comment (kcw-grunt) [inline loafwallet/SupportLFViewModel.swift:2]: Added this to #163

File: loafwallet/TestnetData.swift [hunk 18]
```diff
@@ -1,20 +1,9 @@
-//
```
Comment (kcw-grunt) [inline loafwallet/TestnetData.swift:1]: Need to see if these faucets are still valid and refactor accordingly #166

File: loafwallet/src/Extensions/Async.swift [hunk 19]
```diff
@@ -1,31 +1,22 @@
...
-//  Async.swift
-//  breadwallet
-//
-//  Created by Adrian Corscadden on 2017-03-02.
-//  Copyright © 2017 breadwallet LLC. All rights reserved.
-//
-
 import Foundation
 
 enum Async {
```
Comment (kcw-grunt) [inline loafwallet/src/Extensions/Async.swift:3]: We should check to see if this is ever called: #163

File: loafwallet/src/FeeManager.swift [hunk 20]
```diff
@@ -1,86 +1,79 @@
...
-//  litewallet
-//
-//  Created by Kerry Washington on 2/29/20.
-//  Copyright © 2020 Litecoin Foundation. All rights reserved.
-
-import Foundation
 import FirebaseAnalytics
- 
+import Foundation
+
```
Comment (kcw-grunt) [inline loafwallet/src/FeeManager.swift:3]: @Iferencak this a hack that we need to refactor but its a low priority

File: loafwallet/src/KVStoreCoordinator.swift [hunk 21]
```diff
@@ -1,54 +1,45 @@
...
+		                		let newInfo = WalletInfo(name: $0.walletState.name)
+		                		newInfo.creationDate = $0.walletState.creationDate
+		                		self.set(newInfo)
+		                	}
+		                })
+	}
 
-    private func set(_ info: BRKVStoreObject) {
-        do {
-            let _ = try kvStore.set(info)
```
Comment (kcw-grunt) [inline loafwallet/src/KVStoreCoordinator.swift:45]: Very interesting.
``` 
  _ = ""
  
```

File: loafwallet/src/Strings/it.lproj/Localizable.strings [hunk 22]
```diff
@@ -974,7 +974,7 @@
 "Send.UnstoppableDomains.simpleplaceholder" = "Inserisci dominio";
```
Comment (kcw-grunt) [inline loafwallet/src/Strings/it.lproj/Localizable.strings:975]: @Iferencak is this from a rebase?  Or you speak Italian?  Maybe I did it.? 🤔

File: loafwallet/src/ViewControllers/BiometricsSpendingLimitViewController.swift [hunk 23]
```diff
@@ -1,119 +1,114 @@
...
+	private let limits: [UInt64] = [0, 10_000_000, 100_000_000, 1_000_000_000, 10_000_000_000]
+	private var selectedLimit: UInt64?
+	private var header: UIView?
+	private let amount = UILabel(font: .customMedium(size: 26.0), color: .darkText)
+	private let body = UILabel.wrapping(font: .customBody(size: 13.0), color: .darkText)
 
-    private let cellIdentifier = "CellIdentifier"
-    private let store: Store
-    private let walletManager: WalletManager
-    private let limits: [UInt64] = [0, 10000000, 100000000, 1000000000, 10000000000]
```
Comment (kcw-grunt) [inline loafwallet/src/ViewControllers/BiometricsSpendingLimitViewController.swift:17]: Niiiiice.

File: loafwallet/src/ViewControllers/RootModals/ModalViewController.swift [hunk 24]
```diff
@@ -1,162 +1,159 @@
-//
-//  File.swift
-//  breadwallet
-//
-//  Created by Adrian Corscadden on 2016-11-30.
-//  Copyright © 2016 breadwallet LLC. All rights reserved.
```
Comment (kcw-grunt) [inline loafwallet/src/ViewControllers/RootModals/ModalViewController.swift:6]: There is no way the app is using this file.....er `File.swift` #163

File: loafwallet/src/Views/SearchHeaderView.swift [hunk 25]
```diff
@@ -1,265 +1,263 @@
-//
-//  SearchHeaderView.swift
-//  breadwallet
-//
-//  Created by Adrian Corscadden on 2017-05-02.
-//  Copyright © 2017 breadwallet LLC. All rights reserved.
-//
```
Comment (kcw-grunt) [inline loafwallet/src/Views/SearchHeaderView.swift:7]: Gotta be ded...I removed it years ago: #163

File: loafwallet/src/Views/SeparatorCell.swift [hunk 26]
```diff
@@ -1,27 +1,21 @@
-//
-//  SeparatorCell.swift
-//  breadwallet
-//
-//  Created by Adrian Corscadden on 2017-04-01.
-//  Copyright © 2017 breadwallet LLC. All rights reserved.
-//
-
```
Comment (kcw-grunt) [inline loafwallet/src/Views/SeparatorCell.swift:8]: The Center of Pure Evil 🤮

**Review by kcw-grunt — APPROVED** [review]
My BMI is now under 25%.I have big brain muscles

**Comment by kcw-grunt** [comment]
merging @Iferencak

**Pull request merged**